src/convolution.cpp

```diff
@@ -869,14 +867,14 @@ std::size_t ConvolutionDescriptor::BackwardWeightsGetWorkSpaceSizeWinograd(
     {
         if(ctx.do_search)
             MIOPEN_THROW("Auto-tune is not supported in the get workspace size");
-        const auto ss  = FindWinogradWrWAllSolutions(ctx, {});
-        std::size_t sz = 0;
-        for(const auto& solution : ss)
+        const auto sz_v = FindWinogradWrWWorkspaceSizes(ctx);
```

Comment:
[Recommendation] Use CamelCase for WrW, i.e. `FindWinogradWrwWorkspaceSizes`. Looks weird anyway, but formally valid at least.